File: 2019.03/truffleruby/fibo.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func fibo(n int) int64 {
	if n <= 1 {
		return 1
	}
	return fibo(n-1) + fibo(n-2)
}

func main() {
	n, _ := strconv.Atoi(os.Args[1])
	fmt.Println(fibo(n))
}

File: 2019.03/truffleruby/fibo.java
class fibo{
  private static long f( int n ){
    if (n<=1){
      return 1;
    }
    return f(n-1) + f(n-2);
  }
  public static void main( String[] args ){
    int n = Integer.parseInt(args[0]);
    System.out.println( f(n) );
  }
}